elements/pl-hidden-hints/pl-hidden-hints.py

```diff
@@ -15,22 +18,26 @@ def render(element_html: str, data: pl.QuestionData) -> str:
 
     # Use position so that hints appear in order if show-after-submission values are equal.
     for position, child in enumerate(element):
-        if child.tag == 'pl-hint':
-            pl.check_attribs(child, [], ['show-after-submission'])
+        if child.tag == "pl-hint":
```

Comment:
We expect these diffs to go away once https://github.com/PrairieLearn/PrairieLearn/pull/6793 is merged, correct?

Reply:
Yep! (modulo a merge patch)